codec/src/test/java/io/netty/handler/codec/xml/XmlFrameDecoderTest.java

```diff
@@ -98,38 +96,38 @@ public void testDecodeShortValidXmlWithLeadingWhitespace02() {
 
     @Test
     public void testDecodeShortValidXmlWithLeadingWhitespace02AndTrailingGarbage() {
-        testDecodeWithXml("  \n\r \t<xxx/>\ttrash", "<xxx/>", CorruptedFrameException.class);
+        testDecodeWithXml("  \n\r \t<xxx/>\ttrash", "<xxx/>",
+                CorruptedFrameException.class);
     }
 
     @Test
     public void testDecodeWithCDATABlock() {
-        final String xml = "<book>" +
-                "<![CDATA[K&R, a.k.a. Kernighan & Ritchie]]>" +
-                "</book>";
+        final String xml = "<book>"
+                + "<![CDATA[K&R, a.k.a. Kernighan & Ritchie]]>" + "</book>";
         testDecodeWithXml(xml, xml);
     }
 
     @Test
     public void testDecodeWithCDATABlockContainingNestedUnbalancedXml() {
         // <br> isn't closed, also <a> should have been </a>
-        final String xml = "<info>" +
-                "<![CDATA[Copyright 2012-2013,<br><a href=\"http://www.acme.com\">ACME Inc.<a>]]>" +
-                "</info>";
+        final String xml = "<info>"
+                + "<![CDATA[Copyright 2012-2013,<br><a href=\"http://www.acme.com\">ACME Inc.<a>]]>"
+                + "</info>";
```

Comment:
Could you please keep the original formatting so that the changeset contains only the relevant changes?